docs/sphinx/sections/api/apidocs/internals.rst

```diff
@@ -3,17 +3,19 @@ Internals
 
 .. currentmodule:: dagster
 
-Please note that internal APIs are likely to be in much greater flux pre-1.0 than user-facing APIs,
-particularly if not exported in the top level ``dagster`` module.
+Note that APIs imported from Dagster submodules are not considered stable, and are potentially subject to change in the future.
 
 If you find yourself consulting these docs because you are writing custom components and plug-ins,
 please get in touch with the core team `on our Slack <https://join.slack.com/t/dagster/shared_invite/enQtNjEyNjkzNTA2OTkzLTI0MzdlNjU0ODVhZjQyOTMyMGM1ZDUwZDQ1YjJmYjI3YzExZGViMDI1ZDlkNTY5OThmYWVlOWM1MWVjN2I3NjU>`_.
 We're curious what you're up to, happy to help, excited for new community contributions, and eager
 to make the system as easy to work with as possible -- including for teams who are looking to
 customize it.
 
-Executors
----------
+Executors (Experimental)
+------------------------
+
+APIs for constructing custom executors. While this is considered advanced experimental usage, significant advanced warning will be given before any breakages occur to this set of APIs. Please note that using dagster-provided executors is considered stable, common usage.
```

Comment:
```suggestion
APIs for constructing custom executors. While this is considered advanced experimental usage, significant advanced warning will be given before any breakages occur to this set of APIs. Please note that using Dagster-provided executors is considered stable, common usage.
```

Reply:
I don't think we necessarily need to promise "significant advanced warning". Otherwise, this text looks good.